lib/razorpay.ts

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+import API from './api';
+const pkg = require('../package.json')
+import {
+  validateWebhookSignature
+} from './utils/razorpay-utils';
+
+import { Headers } from './types';
+
+import Payments from './resources/payments';
+import Refunds from './resources/refunds';
+import Orders from './resources/orders';
+import Customers from './resources/customers';
+import Transfers from './resources/transfers';
+import VirtualAccounts from './resources/virtualAccounts';
+import Invoices from './resources/invoices';
+import Plans from './resources/plans';
+import Subscriptions from './resources/subscriptions';
+import Addons from './resources/addons';
+
+class Razorpay {
+  static VERSION = pkg.version
+
+  key_id: string;
+  key_secret: string;
+  api: InstanceType<typeof API>;
+
+  static validateWebhookSignature (body, signature, secret) {
```

Comment:
Add types for arguments?

Reply:
Or is it inferred from `validateWebhookSignature`?

Reply:
Will do in the second pass. Second pass removed the implicit any flag. Changes in this PR is with the implicit any flag and resolving errors that still occur.